app/components/agendapunt-overview-item.hbs

```diff
@@ -2,16 +2,14 @@
   <div class="au-c-card__header">
     <div class="au-o-grid au-o-grid--small">
       <div class="au-o-grid__item au-u-3-4@medium">
-        <AuHeading @level="2" @skin="4" property="dct:title">{{clean-spaces @agendapunt.titel}}</AuHeading>
+        <h2 class="au-c-heading--4" property="dct:title">{{clean-spaces @agendapunt.titel}}</</h2>
         {{#if @agendapunt.type}}
           <span property="besluit:Agendapunt.type" style='display: none;' resource="{{@agendapunt.type.uri}}" typeof="skos:Concept">{{@agendapunt.type.label}}</span>
         {{/if}}
         {{#if @agendapunt.beschrijving}}
-          <p property="dct:description">
-            <AuHelpText @skin="secondary">{{clean-spaces @agendapunt.beschrijving}}</AuHelpText>
-          </p>
+          <p property="dct:description"><span class="au-c-help-text--secondary">{{clean-spaces @agendapunt.beschrijving}}</span></p>
         {{/if}}
-        <span property="besluit:geplandOpenbaar" content={{@agendapunt.geplandOpenbaar}} class="au-u-hidden"></span>
+        <span property="besluit:geplandOpenbaar" content="{{@agendapunt.geplandOpenbaar}}" class="au-u-hidden"></span>
```

Comment:
wrapped in quotes so "true" and "false" render correctly

Reply:
Hmm, I'm not sure I like that. It looks like a mistake. I would explicitly set the string versions manually:

`content="{{if @agendapunt.geplandOpenbaar "true" "false"}}`

It's more code but there is no confusion that this is the wanted behavior.

Reply:
I guess the fact that I felt the need to add the comment proves your point :)